ps_shoppingcart.js

```diff
@@ -38,6 +38,7 @@ $(document).ready(function () {
 
         if (event && event.reason) {
           requestData = {
+            id_customization: $('#product_customization_id').val(),
```

Comment:
It seems we still need to modify that line before validating the PR, doesn't it?